client/home/lib/stacks/homestacksimport.coffee

```diff
@@ -0,0 +1,151 @@
+kd = require 'kd'
+Encoder = require 'htmlencode'
+EnvironmentFlux = require 'app/flux/environment'
+
+remote = require('app/remote').getInstance()
+showError = require 'app/util/showError'
+OutputView = require 'stacks/views/stacks/outputview'
+
+getRepoFromQuery = (query) ->
+
+  { repo, branch } = query
+
+  repo = Encoder.XSSEncode repo
+
+  if branch
+    branch = Encoder.XSSEncode branch
+    repo = "#{repo}/#{branch}"  if branch
+
+  return repo
+
+
+module.exports = class HomeStacksImport extends kd.CustomHTMLView
+
+  constructor: (options = {}, data) ->
+
+    options.cssClass = 'HomeAppView-Stacks--import'
+
+    super options, data
+
+
+  viewAppended: ->
+
+    @addSubView new kd.CustomHTMLView
+      tagName: 'h2'
+      partial: "Stack Importer"
```

Comment:
The files under `home/stacks` are being [ignored by coffeelint](https://github.com/koding/koding/blob/master/.coffeelintignore#L50).